src/LegacyCallHandler.tsx

```diff
@@ -457,6 +457,10 @@ export default class LegacyCallHandler extends EventEmitter {
         logger.debug(`${logPrefix} paused audio`);
     }
 
+    public isPlaying(audioId: AudioID): boolean {
+        return !!this.playingSources[audioId];
+    }
```

Comment:
Not loving the lack of documentation on this, but I guess it fits the general vibe in this class.

In particular, what happens if you have a media which doesn't loop, but has reached its end without being explicitly `pause`d? Does this return `true` or `false`?

Reply:
Good point, iterated